### PR TITLE
feat(screenshot-list): show flaky indicator on hover

### PR DESCRIPTION
**Pull request by jsfez**

### CODE REVIEW

File: apps/app/src/ui/IconButton.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const IconButton = forwardRef<HTMLButtonElement, IconButtonProps>(
         as="button"
         className={clsx(
           variantClassName,
-          "flex cursor-default items-center rounded-lg border border-transparent p-[7px] text-sm text-icon-button-on transition disabled:opacity-70 [&>*]:h-4 [&>*]:w-4"
+          "flex cursor-default items-center self-center rounded-lg border border-transparent p-[7px] text-sm text-icon-button-on transition disabled:opacity-70 [&>*]:h-4 [&>*]:w-4"
```
Comment (gregberge) [inline apps/app/src/ui/IconButton.tsx:35]:
```suggestion
          "flex cursor-default items-center rounded-lg border border-transparent p-[7px] text-sm text-icon-button-on transition disabled:opacity-70 [&>*]:h-4 [&>*]:w-4"
```

Self positioning should not be part of component, like margin.

File: apps/app/src/ui/FlakyIndicator.tsx [hunk 2]
```diff
@@ -3,23 +3,24 @@ import { BugAntIcon } from "@heroicons/react/20/solid";
...
+  "This screenshot comparison has high instability in recent builds and may be a flaky.";
+
+export const FlakySuspectedIcon = () => {
   return (
-    <MagicTooltip tooltip="Move to end for flaky suspicion">
-      <Icon
-        className={`absolute top-3 right-4 z-30 h-4 w-4 text-${color}-400`}
-      />
+    <MagicTooltip tooltip={flakySuspectedTooltip}>
+      <FlakyIcon className={`text-${flakySuspectedColor}-400`} />
```
Comment (gregberge) [inline apps/app/src/ui/FlakyIndicator.tsx:15]:
```suggestion
      <FlakyIcon className={`text-${flakySuspectedColor}-400`} />
```

Break Tailwind.

**Review by gregberge — CHANGES_REQUESTED** [review]

File: apps/app/src/ui/FlakyIndicator.tsx (hunk 2 above)
Comment (jsfez) [reply apps/app/src/ui/FlakyIndicator.tsx:15]: There is no changes in your suggestion

Comment (gregberge) [reply apps/app/src/ui/FlakyIndicator.tsx:15]: No I will not code it right for you 😬

Comment (gregberge) [reply apps/app/src/ui/FlakyIndicator.tsx:15]: Classes can't be dynamic.

**Pull request merged**